react-for-atom.js

```diff
@@ -10,21 +10,37 @@ if (typeof atom.__DO_NOT_ACCESS_React_Singleton === 'undefined') {
     // circumvent React Dev Tools console warning
     window.__REACT_DEVTOOLS_GLOBAL_HOOK__ = {};
   }
-  var exportables = {
-    default: function() { return require('react'); },
-    createFragment: function() { return require('react-addons-create-fragment'); },
-    CSSTransitionGroup: function() { return require('react-addons-css-transition-group'); },
-    LinkedStateMixin: function() { return require('react-addons-linked-state-mixin'); },
-    Perf: function() { return require('react-addons-perf'); },
-    PureRenderMixin: function() { return require('react-addons-pure-render-mixin'); },
-    React: function() { return require('react'); },
-    ReactDOM: function() { return require('react-dom'); },
-    ReactDOMServer: function() { return require('react-dom/server'); },
-    shallowCompare: function() { return require('react-addons-shallow-compare'); },
-    TestUtils: function() { return require('react-addons-test-utils'); },
-    TransitionGroup: function() { return require('react-addons-transition-group'); },
-    update: function() { return require('react-addons-update'); },
-  };
+  var exportables = process.env.NODE_ENV !== 'production' ?
+    {
+      default: function() { return require('react'); },
+      createFragment: function() { return require('react-addons-create-fragment'); },
+      CSSTransitionGroup: function() { return require('react-addons-css-transition-group'); },
+      LinkedStateMixin: function() { return require('react-addons-linked-state-mixin'); },
+      Perf: function() { return require('react-addons-perf'); },
+      PureRenderMixin: function() { return require('react-addons-pure-render-mixin'); },
+      React: function() { return require('react'); },
+      ReactDOM: function() { return require('react-dom'); },
+      ReactDOMServer: function() { return require('react-dom/server'); },
+      shallowCompare: function() { return require('react-addons-shallow-compare'); },
+      TestUtils: function() { return require('react-addons-test-utils'); },
+      TransitionGroup: function() { return require('react-addons-transition-group'); },
```

Comment:
Why not be consistent and use `require('react/dist/react-with-addons')` for all of these ones? I hear that basically nobody is running in dev mode anyway so it probably doesn't matter, but you'd get some speedups here too (cuts out the runtime `NODE_ENV` checks). All of this might not be supported for too much longer if we ever get flat bundling working but you'd have to change everything then anyway.